### ISSUE TITLE
Accessibility suggestions for figure and figure caption

### ISSUE THREAD

**Issue by jooyoungseo** [jgm/pandoc#6782]
As a blind person who uses screen reading software, I feel Pandoc's support and its syntax for alt text and caption are subject to further discussion.

`title` attribute is not recommended to use for `<img>` because its value can be not read when alt attribute is present (depending on screen reader settings, etc.): see [this posting for more info](https://developer.paciellogroup.com/blog/2012/01/html5-accessibility-chops-title-attribute-use-and-abuse/#:~:text=When%20to%20use%20the%20title,be%20important%20to%20any%20user.).

I think the following would be more logical:

`![alt here](foo.png "caption text here")`

### html

1. When both alt and caption are provided

Please do not use `aria-hidden = "true"` when figcaption is visually available on the screen. Screen-reader users should have the same information if possible (see [this spec](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Figure_Role)):

``` html
<figure>
<img src = "foo.png" alt = "alt text here"/>
<figcaption>caption here</figcaption>
</figure>
```

2. Only when alt is provided

There is no need to visually display figcaption when it is not provided:

``` html
<figure>
<img src = "foo.png" alt = "alt text here"/>
</figure>
```

Or,

``` html
<figure>
<img src = "foo.png" alt = "alt text here"/>
<figcaption aria-hidden = "true"></figcaption>
</figure>
```

3. Only when caption is provided:

Since `alt` text is more important than figcaption (accessibility wise), we can pass figcaption to alt:

``` html
<figure>
<img src = "foo.png" alt = "caption here"/>
<figcaption>caption here</figcaption>
</figure>
```

4. Neither alt nor caption is provided.

``` html
<figure>
<img src = "foo.png" alt = ""/>
</figure>
```

Or,

``` html
<figure>
<img src = "foo.png" alt = ""/>
<figcaption aria-hidden = "true"></figcaption>
</figure>
```

### docx

1. When both alt and caption are provided

"alt here" goes to image title description; "caption here" is inserted as a visible text under the figure.

Currently, Pandoc's behavior is the other way around. Again, the proposed syntax is as follows:

`![alt text](foo.png "visible text caption below")`

2. Only when alt is provided

There is no need to visually present caption text; however, still need to keep given alt text.

3. Only when caption is provided:

Since `alt` text is more important than figcaption (accessibility wise), we should pass figcaption to alt while visually presenting the given caption text.

4. Neither alt nor caption is provided.

Very rare case, but we need to pass alt as "" with no visible caption.

**Comment by mb21** [comment]
Looks like a duplicate of https://github.com/jgm/pandoc/issues/4737, which was closed in favour of https://github.com/jgm/pandoc/issues/3177 ?

**Comment by jgm** [comment]
Yes.  There's a simple reason we use alt rather than title for the caption:  the title is a plain string; only the part between `[]` can have formatting.  But the situation is less than ideal, as noted in #3177 and #4737.

**Comment by brunoprietog** [comment]
As a blind user, I agree with this, it would be great if they could implement it. Especially because caption and alt text are not the same, they have different purposes.

**Comment by tarleb** [comment]
I'm closing this in favor of #3177, but added an explicit note to that issue so we do not loose the valuable info here.
